### DISCUSSION TITLE
How do you paste multi lines?

### DISCUSSION THREAD

**Discussion by jberent** [JanDeDobbeleer/oh-my-posh#2891] [Troubleshoot]
pre oh-my-posh I could paste in multiple lines, e.g., a new function I am testing. This does not work anymore with pwsh-7 and pwsh-windows, even if I disable `function:prompt` so I guess it has something to do with prompt redraw (??)  If the paste is long enough (< 20 lines) the console gets real messed up writing the lines out slowly until:

```
Exception:
System.ArgumentOutOfRangeException: The value must be greater than or equal to zero and less than the console's buffer size in that dimension.
```
Oops, something went wrong.  Please report this bug with the details below.
Report on GitHub: https://github.com/lzybkr/PSReadLine/issues/new

**Comment by JanDeDobbeleer** [comment]
@jberent did you update PSReadLine? Because that one's responsible for this.

**Comment by jberent** [comment]
Thanks, I thought I had a brand new installation but 2.1.0 was installed. Works now with 2.2.6

Sent via the Samsung Galaxy S21 Ultra 5G, an AT&T 5G smartphone
Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
From: Jan De Dobbeleer ***@***.***>
Sent: Wednesday, October 5, 2022 4:48:21 PM
To: JanDeDobbeleer/oh-my-posh ***@***.***>
Cc: Jonathan Berent ***@***.***>; Mention ***@***.***>
Subject: [External] Re: [JanDeDobbeleer/oh-my-posh] How do you paste multi lines? (Discussion #2891)


@jberent<https://github.com/jberent> did you update PSReadLine? Because that one's responsible for this.
